src/app/ui/ui-select/ui-select.component.ts

```diff
@@ -72,6 +72,9 @@ export class UiSelectComponent implements OnInit {
     if (this.dropdownList) {
       this.dropdownList.nativeElement.scrollTop = 0;
       this.setScrollMax();
+      if (this.open) {
```

Comment:
It looks like this is left over from something?

Reply:
whooops